README.md

```diff
@@ -13,3 +14,38 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+## My Response
+
+1.
+The first is that when we are analyzing an algorithm we ignore any constants that might arise, this can change things significantly.  On a lower level, $x^2$ and $4x^2$ are not the same function even though the would have the same complexity, $4x^2$ increases faster than its parent function.
+
+The second is that the complexity of an algorithm doesn't take into account the hardware of the machine running the program.  It could run faster or slower based on the resources available.
+
+Cade brought up this one while I was talking to him in lab.  The bounds of our complexity can come into question when we start looking at Big Theta complexity.  For instance the $\Theta$ complexity of a program might be $nlogn$ but its upper and lower are $2^n$ and $n$ respectively, which are two very different functions that leave a lot of room for movement between them.
```

Comment:
...that sounds like you're not talking about $\Theta$? At least $\Theta$ is not misleading in this respect.

Reply:
I think I understand, the upper and lower bound still apply to this idea but I shouldn't be bringing in Theta complexity for this case?